Update Doc Edge Caching - Slice (#79)
aziontech/docs_en: edge-caching/index.md

---

## 6. Slice Settings {#slice-settings}

> Slice is an Edge Application feature that allows the processing of large amounts of data in a more effective way, reducing latency and saving bandwidth. By activating this feature, the file or media itself is reduced to small pieces. Such fragments are gradually delivered to the end-user according to the data consumption, avoiding a transfer rush that might not be finalized by the end-user. The data is cached on demand whenever the user requires it.

Author: Ok! :)

> Slice is not only restricted to Azion's Edge Nodes. It is possible to apply the feature on L2 modules, which allows data to be cached longer, delivering on-demand data to the end-user, increasing performance and availability.

Reviewer: ```suggestion
Slice is not only restricted to Azion's Edge Nodes. It is possible to apply the feature on L2 modules, which allows content to be cached longer, delivering on-demand content to the end-user, increasing performance and availability.
```

---

## 6. Slice Settings {#slice-settings}

> Slice is an Edge Application feature that allows the processing of large amounts of data in a more effective way, reducing latency and saving bandwidth. By activating this feature, the file or media itself is reduced to small pieces. Such fragments are gradually delivered to the end-user according to the data consumption, avoiding a transfer rush that might not be finalized by the end-user. The data is cached on demand whenever the user requires it. When the request is established, Slice begins to operate.

Author: ```suggestion
Slice is an Edge Application feature that allows the processing of large amounts of data in a more effective way, reducing latency and saving bandwidth. By activating this feature, the file or media itself is reduced to small pieces. Such fragments are gradually delivered to the end-user according to the data consumption, avoiding a transfer rush that the end-user might not finalize. The data is cached on demand whenever the user requires it. When the request is established, Slice begins to operate.
```

> The feature operates with a standard of 1024kb slice-range of content. In case you'd like to change that slice range, make sure to contact our [Sales Team](https://www.azion.com/en/contact-sales/) for a custom dimension.

Author: ```suggestion
The feature operates with a standard of 1024kB slice-range. In case you'd like to change that slice range, make sure to contact our [Sales Team](https://www.azion.com/en/contact-sales/) for a custom dimension.
```

> Slice is not only restricted to Azion's Edge Nodes. It is possible to apply the feature on L2 modules, which allows content to be cached longer, delivering on-demand content to the end-user, increasing performance and availability.

Reviewer 2: ```suggestion
Slice is not only restricted to Edge Caching module. It is possible to apply the feature on L2 modules, which allows content to be cached longer, delivering on-demand content to the end-user, increasing performance and availability.
```

> Slice is not only restricted to Edge Caching module. It is possible to apply the feature on L2 modules, which allows content to be cached longer, delivering on-demand content to the end-user, increasing performance and availability.

Author: ```suggestion
Slice is not only restricted to Edge Caching module. It is possible to apply the feature on L2 module, which allows content to be cached longer, delivering on-demand content to the end-user, increasing performance and availability.
```
